Fix an issue building Nautilus with libportal-0.5+
where expected header is moved to different directory
in staging sysroot

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
--- a/meson.build
+++ b/meson.build
@@ -123,7 +123,7 @@ gtk = dependency('gtk+-3.0', version: '>
 libhandy = dependency('libhandy-1', version: '>= 1.1.90')
 libportal = []
 if get_option('libportal')
-  libportal = dependency('libportal', version: '>= 0.3')
+  libportal = dependency('libportal-gtk3', version: '>= 0.3')
 endif
 selinux = []
 if get_option('selinux')
--- a/src/nautilus-files-view.c
+++ b/src/nautilus-files-view.c
@@ -92,7 +92,7 @@
 
 #ifdef HAVE_LIBPORTAL
 #include <libportal/portal.h>
-#include <libportal/portal-gtk3.h>
+#include <libportal-gtk3/portal-gtk3.h>
 #endif
 
 /* Minimum starting update inverval */
